.github/workflows/publish-HOWTOs.yml

```diff
@@ -0,0 +1,86 @@
+name: Publish HOWTOs
+on:
+  - push
+  - workflow_dispatch
+jobs:
+  build:
+    name: Build and prepare HOWTOs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: npm install
+      - name: Build packages
+        run: npm run build
+      - name: Prepare for publishing (GitHub Pages)
+        run: npm run prep
+        env:
+          PKG_HOWTOS_LOCATION: github
+          PKG_HOWTOS_PATH: ${{ github.ref_name }}
```

Comment:
We would extract just the branch name without refs/heads/ it looks like this is using the full path as the target folder?

Reply:
From https://docs.github.com/en/actions/learn-github-actions/contexts#github-context:
> github.ref_name | string | The branch or tag name that triggered the workflow run.
> -- | -- | --

What you're describing is:
> 

> github.ref | string | The branch or tag ref that triggered the workflow run. For branches this is the format `refs/heads/<branch_name>`, and for tags it is `refs/tags/<tag_name>`.
> -- | -- | --

Reply:
Please see the actual deployment done from this branch (`sad/DEVOP/71`) to both `aws s3` and `github pages`: https://github.com/built-on-openfin/workspace-starter/pull/105#issuecomment-1136156764